schema/mysql/v57/temporal/versioned/v1.0/schema.sql

```diff
@@ -227,4 +222,42 @@ CREATE TABLE history_tree (
   data           BLOB NOT NULL,
   data_encoding  VARCHAR(16) NOT NULL,
   PRIMARY KEY (shard_id, tree_id, branch_id)
+);
+
+CREATE TABLE queue (
```

Comment:
What about `replication_tasks_dlq`?  This was introduced in v0.3 but I don't see it ported in collapsed schema.

Reply:
Actually found it.  Merge looked weird.